.github/actions/dockerfiles/Dockerfile.alpine-binary

```diff
@@ -10,10 +10,9 @@ ARG TARGETVARIANT
 ARG REPO
 
 RUN case ${TARGETPLATFORM} in \
-    linux/amd64/v2) BIN_ARCH=linux-glibc-x64-v2 ;; \
```

Comment:
What's the context behind this platform being removed?

Reply:
this change came from when i was confirming the images built in @BowTiedDevOps fork - what i noticed is that the alpine image is pulling in the glibc binaries (and has been for a few months - the alpine image either needs these changes, or needs to be removed since it's not being used). 

the `v2` variant is also not being built anymore (v3 is the default), so that match will never happen. 
it's possible we could build for that CPU in the future - but i would think that's unlikely since the arch is from 2008, and we haven't produced binaries for that arch in a long time. 


https://github.com/stacks-network/stacks-core/blob/develop/.github/workflows/create-source-binary.yml#L42